pkg/build/generator/generator_test.go

```diff
@@ -741,7 +741,7 @@ func TestGenerateBuildFromConfig(t *testing.T) {
 	bc := &buildapi.BuildConfig{
 		ObjectMeta: kapi.ObjectMeta{
 			Name:      "test-build-config",
-			Namespace: "test-namespace",
+			Namespace: kapi.NamespaceDefault,
```

Comment:
also that still doesn't explain why this object which had a valid namespace, is being changed to use a different namespace?